### ISSUE TITLE
Is this a Delphi based app?

### ISSUE THREAD

**Issue by vonHabsi** [gdiJS/base#1]
It looks like a Delphi project.

Will it work with FreePascal and are there plans to get it work on LInux?

**Comment by PsyChip** [comment]
yes, i compiled the wrapper part in rad studio 12. technically it would compile on fpc too with couple days of work. there is no plans for other platforms but contributions are welcome.

**Comment by code-kungfu** [comment]
@vonHabsi  You can download the free Community Edition of Delphi 12.1 and build the application just fine.

Looking over the code, there's a lot of Windows specific stuff going on so getting it to run on Linux or macOS would take some time to abstract and separate the layers.
